src/main/java/org/isf/utils/file/FileTools.java

```diff
@@ -129,7 +129,7 @@ public static Date getTimestamp(File file) {
 	 * @return the list of retrieved date (first null)
 	 */
 	public static List<Date> getTimestampFromName(String formattedString) {
-		List<Date> datesFound = new ArrayList<Date>();
+		List<Date> datesFound = new ArrayList<>(1);
```

Comment:
Because static checkers complain about array allocations without sizes as they typically either allocate too many or too few.  In this case we know that only one timestamp is returned.    This was the case where we were going to call the method signature to return a single Date but the changes rippled too far.

Reply:
yes I remember... that's why I dont' remember why we changed the number of returned result but kept the List type, probably to not change the method's signature.
Alright then. Thanks!